app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class='default-font'>
+  <%= link_to category_url(category) do %>
+    <li >
+      <div class='m-3 p-4 card is-flex is-justify-content-space-between'>
+        <div class='image'>
+          <%= category.icon %>
+        </div>
```

Comment:
- [ ] In the category view element you are just rendering the icon value as a raw string, and when a user enters a huge size string, it messes up the UI. In my opinion, using the raw string as an icon is not a good idea, but if you chose that it should be like that you should at least validate the minimum string size that should be input in the form. I also suggest that you have an `img` tag for the icon which has the `src` property value equal to an image address URL, a user can input. 

![image](https://user-images.githubusercontent.com/64894927/143424628-4feb1a54-9681-4050-aacd-4001738715ca.png)